src/infra/src/table/migration/m20250109_085148_alter_dashboards_folder_id.rs

```diff
@@ -0,0 +1,207 @@
+// Copyright 2024 OpenObserve Inc.
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+//! Drops the folder's created_at column since Sea ORM maps this column's data
+//! type to different Rust types creating runtime errors when running on a
+//! non-PostgreSQL database.
+
+use sea_orm::{
+    ActiveModelTrait, ColumnTrait, EntityTrait, PaginatorTrait, QueryFilter, Set, TransactionTrait,
+};
+use sea_orm_migration::prelude::*;
+
+const DASHBOARDS_FOLDERS_FK: &str = "dashboards_folders_fk";
+
+#[derive(DeriveMigrationName)]
+pub struct Migration;
+
+#[async_trait::async_trait]
+impl MigrationTrait for Migration {
+    async fn up(&self, manager: &SchemaManager) -> Result<(), DbErr> {
+        add_nullable_folder_uuid_column(manager).await?;
+        update_folder_uuid_column(manager).await?;
+        drop_foreign_key(manager).await?;
+        delete_folder_id_column(manager).await?;
+        rename_folder_uuid_column(manager).await?;
+        Ok(())
+    }
+
+    async fn down(&self, _manager: &SchemaManager) -> Result<(), DbErr> {
+        // Reversing this migration is not supported.
+        Ok(())
+    }
+}
+
+// Removes the old created_at column.
+async fn add_nullable_folder_uuid_column(manager: &SchemaManager<'_>) -> Result<(), DbErr> {
+    manager
+        .alter_table(
+            Table::alter()
+                .table(Dashboards::Table)
+                .add_column(
+                    ColumnDef::new(Dashboards::FolderUuid)
+                        .string_len(256)
```

Comment:
```suggestion
                    ColumnDef::new(Dashboards::FolderUuid)
                        .char_len(27)
```

These KSUID strings should always be 27-character fixed length strings so we can make the column type a bit more restrictive.